workshops/views.py

```diff
@@ -441,11 +442,33 @@ class EventCreate(LoginRequiredMixin, CreateViewContext):
     model = Event
     fields = '__all__'
 
+@login_required
+def event_edit(request, event_ident):
+    event_form_factory = modelformset_factory(Event, fields='__all__', extra=0)
+    task_form_factory = inlineformset_factory(Event, Task, fields='__all__', extra=1, can_delete=False)
```

Comment:
I'm not certain about these two form factories. Wouldn't it be better to have ModelForms explicitely implemented in `workshops.forms`?